client/me/security-2fa-disable/index.jsx

```diff
@@ -96,7 +96,7 @@ class Security2faDisable extends Component {
 			<div>
 				<p>
 					{ translate(
-						"You've enabled Two-Step Authentication on your account — smart move! " +
+						"You've enabled two-step authentication on your account — smart move! " +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'You've enabled Two-Step Authentication on your account — smart move! When you log in to WordPress.com, you'll need to enter your username and password, as well as a unique passcode generated by an app on your mobile device.' )` *ES Score: 10*